docs/overview/README.md

```diff
@@ -1,18 +1,27 @@
 
-## IOTA client library in a nutshell
-The library provides two types of API calls under a common interface:
-* `Full node API calls`: those calls are basically translated to native node rest api calls. For more information, please see [node rest API reference](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)
-* `General high level API calls`: those are convenience functions with some typical default parameters in order to use them in a straightforward manner. They typically implement several recommended steps for the given task. Implementation details are part of the full specification
+This approach makes iota.rs easier for you to use and understand, and for us to develop and maintain. But since you are in full control of the data management, you also fully responsible for it. It could feel tedious or overwhelming if you have to manage complex or sensitive data.
 
-See the full specification [here](../specs/).
+If you plan on managing funds in your application, take a look at our [wallet.rs](https://wallet-lib.docs.iota.org/) library instead. It already includes our best security practices and can build and store wallets and manage token transactions. The wallet.rs library uses iota.rs to communicate with the IOTA network, and [stronghold.rs](https://stronghold.docs.iota.org/) to store sensitive data. Unlike iota.rs, it has a state.
 
-## High level layered overview:
-![iota layers overview](layered_overview.svg)
\ No newline at end of file
+## Supported Languages
+
+We have implemented the iota.rs library in Rust and prepared bindings for JavaScript and Python. The API stays the same no matter the language you choose, as we aim to make our libraries equally simple to use across all supported languages. You can read more on reasoning behind this design in this [blog post](https://blog.iota.org/the-new-iota-client-libraries-harder-better-faster-stronger/).
+
+## Your Application In the IOTA Network
+
+Your application communicates with iota.rs either directly or through a node.js or python binding. The iota.rs library turns your requests into REST API calls and sends them to a node through the Internet. The node, in turn, interacts with the rest of an IOTA network, which could be the main operational network (mainnet) or a network for testing purposes (devnet).
+
+Different nodes could run on a different software, but they always expose the same interface to clients. For example, one node could be a [Hornet](https://hornet.docs.iota.org/) node and the other could be a [Bee](https://bee.docs.iota.org/) node, and they both would appear the same for any client.
+
+![An illustration for the text above.](./layered_overview.svg "An overview of IOTA layers.")
```

Comment:
Perhaps we could replace "An illustration for the text above" with "An overview of IOTA layers." as it's more descriptive.